src/actions/UpdateApiActions.js

```diff
@@ -109,6 +109,24 @@ export default function UpdateApiActions(
         }
     });
 
+    this.postByHref = (href, body = '') => ({
```

Comment:
I don't think you need the default for body, if a null comes in that's fine

Reply:
I was playing with JSON.stringify in the console and it makes null 'null', but an empty string '' just becomes "''" which I preferred, probably doesn't really matter though no